test/models/micropost_test.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class MicropostTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
Add model tests. See IO and DM for examples. Form is `should succeed validation`